schemas/test/en/test_calculated_summary_dependent_questions.json

```diff
@@ -0,0 +1,193 @@
+                                        "label": "Capital expenditure",
+                                        "description": "Enter the full value",
+                                        "minimum": {
+                                            "value": 0,
+                                            "exclusive": true
+                                        },
+                                        "maximum": {
+                                            "value": {
+                                                "identifier": "answer-1",
+                                                "source": "answers"
+                                            },
+                                            "exclusive": false
+                                        },
+                                        "decimal_places": 2,
+                                        "currency": "GBP"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "id": "block-3",
+                            "type": "Question",
+                            "question": {
+                                "id": "question-3",
+                                "title": "How much did you spend on clothes?",
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "id": "answer-3",
+                                        "mandatory": true,
+                                        "type": "Currency",
+                                        "label": "Capital expenditure",
+                                        "description": "Enter the full value",
+                                        "minimum": {
+                                            "value": 0,
+                                            "exclusive": true
+                                        },
+                                        "decimal_places": 2,
+                                        "currency": "GBP"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "id": "block-4",
+                            "type": "Question",
+                            "question": {
+                                "id": "question-4",
+                                "title": "Of the money spent on clothes, how much did you spend on shoes?",
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "id": "answer-4",
+                                        "mandatory": true,
+                                        "type": "Currency",
+                                        "label": "Capital expenditure",
```

Comment:
These all need to be updated, the schema doesn't quite make sense. Even test schema should be easy to digest. They all have the same labels, so Calculated summary looks odd.

Reply:
Updated the labels now 👍